### DISCUSSION TITLE
Is it possible to running tenant on set of drives or set of nodes?

### DISCUSSION THREAD

**Discussion by tanfpt** [minio/minio#15561] [Q&A] [answered]
Hello,
I'm building minio on k8s, and want to find out how to run each tenant on set of drives or nodes, since afaik minio don't have QoS mechanism. Is it possible? Do we have any documentation about this?
Thank you,
Tan

**Comment by harshavardhana** [comment, marked as answer]
Just follow our operator documentation @tanfpt
